src/traces/sankey/attributes.js

```diff
@@ -168,6 +168,14 @@ var attrs = module.exports = overrideAll({
     },
 
     link: {
+        arrowlen: {
+            valType: 'number',
+            min: 0,
+            dflt: 0,
+            description: [
+                'Sets the length (in px) of the links arrow, if 0 no arrow will be drawn.'
```

Comment:
In addition to pixel, having a constant ratio between width of the links may be of interest. 
For example 0.5 could be used to describe an arrow with 45 degrees angle (90 degrees at the top).
One idea can be to use values less than one as ratios and those greater than one as pixels.
@alexcjohnson your thoughts on this :pray:

Reply:
Interesting idea @archmoj - I can certainly see people wanting a constant angle, though it has the disadvantage that two flows into the same node will have the backs of their arrowheads misaligned, which could in turn cause the links to start to overlap (for example two parallel links shifting sharply upward, the bottom one bigger than the top)

So this is going to be a bit more complicated to implement... let's ignore it for now, if we do add it down the line I think it might be more intuitive as a new attribute, perhaps `arrowangle`? Maybe they could even work together, if you give `arrowangle` you draw lines at that angle, but if they make it to `arrowlen` without reaching the edge then you go straight up & down... that way we can avoid much of the misalignment and all the potential overlap.